tests/acceptance/28_inform_testing/01_files/create.cf.expected

```diff
@@ -0,0 +1,2 @@
+    info: Created file '/tmp/TEST.cfengine', mode 0600
+    info: files promise '/tmp/TEST.cfengine' repaired
```

Comment:
```
info: files promise '/tmp/TEST.cfengine' repaired, create with mode 0600
```

Not sure about mentioning `mode 0600` since that is not specified in the policy.